dataactbroker/handlers/fileHandler.py

```diff
@@ -472,9 +472,13 @@ def checkD1File(self, submission_id=None):
             if result.status_code != 200:
                 raise ResponseException(result.data)
 
-        url = "" if d1_file.url is None else d1_file.url
-        error_message = "" if d1_file.error_message is None else d1_file.error_message
+        user_id = LoginSession.getName(session)
+        slash_index = d1_file.upload_file_name.rfind("/")
+        timestamped_filename = d1_file.upload_file_name[slash_index + 1:]
```

Comment:
Nothing worth holding this PR back, but this might be a little cleaner:

```python
timestamped_filename = d1_file.upload_file_name.split('/')[-1]
```